tests/mocks_test.py

```diff
@@ -226,3 +232,44 @@ def run():
 
     run()
     assert_reset()
+
+
+def test_import_layout(mocker):
```

Comment:
please add docs to all of the tests lice we do in content and sdk Its

Reply:
done